OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1316,6 +1337,32 @@ public void TestGetFeatureVariableStringReturnsCorrectValue()
             Assert.Null(OptimizelyMock.Object.GetFeatureVariableString(featureKey, variableKeyNull, TestUserId, null));
         }
 
+        [Test]
+        public void TestGetFeatureVariableJSONReturnsCorrectValue()
+        {
+            var featureKey = "featureKey";
+            var variableKeyString = "varJSONString1";
+            var variableKeyIntString = "varJSONString2";
+            var variableKeyDouble = "varJSONDouble";
+            var variableKeyNull = "varNull";
+            var featureVariableType = "json";
+
+            OptimizelyMock.Setup(om => om.GetFeatureVariableValueForType<OptimizelyJson>(It.IsAny<string>(), variableKeyString, It.IsAny<string>(),
+                It.IsAny<UserAttributes>(), featureVariableType)).Returns(new OptimizelyJson("{\"string\": \"Test String\"}", ErrorHandlerMock.Object, LoggerMock.Object));
```

Comment:
take this in a dictionary and use the same approach. comparedata.
`{\"string\": \"Test String\"}"`

Reply:
Please do it same thing for all.